<!-- This file needs to be edited by the lab developer to suit
the requirements of their lab in particular.-->

<!-- Add class="default" to include any element as it is
specified in default.html. 
Do not include class="default" to the elements that you want to
edit -->

<!DOCTYPE html>
<html>
<head></head>
<body>

<div id="experiment"> <!-- The Experiment Document Container-->

  <!-- The lab Header contains the logo and the name of the lab,
  usually displayed on the top of the page-->

  <header id="experiment-header" class="default">
  
    <div id="experiment-header-logo" class="logo">
      <!-- Enclose the logo image of your lab or write it in 
      text-->
      <img src="../images/logo.jpg" />
    </div>

    <div id="experiment-header-heading" class="heading">
      <!-- Write the name of your lab and link it to the home 
      page of your lab (h1 tag is preferred while writing your 
      lab name)-->
      <a href="../index.html">Circular Dichroism Virtual Lab</a>	
    </div>

    <!-- Add any additional element you want to add to the lab 
    header, For example : Help (Enclosing them with suitable 
    div is recommended)-->

  </header>


  <!-- The lab article is the main content area where all the 
  experiment content sits-->
  <article id="experiment-article">
  
    <!-- The lab article has an header, optional navigational 
    menu, number of sections, an optional sidebar and a closing 
    footer-->
     <div id="experiment-article-breadcrumb" class="breadcrumb">
     </div>
    
      <header id="experiment-article-heading" class="heading">
        <!-- You can add a welcome message or title of the 
        experiment here -->
        To Study The Effect of Chemical Denaturants on Protein Stability.
        <!-- Add any additional element if required with proper 
        enclosing-->
      </header>

      <!-- Navigation menu is useful to organize the view of 
      multiple sections inside the article-->
      <nav id="experiment-article-navigation" class="default">
        <ul id="experiment-article-navigation-menu">
          <!-- The menu can be dynamically generated to contain 
          the headings of your sections or instead write the 
          menu items of your choice individually enclosedu in 
          <li> tag as shown below-->
        </ul>
      </nav>

      <!-- All the sections of your lab or experiment can be 
      enclosed together with a div element as shown below-->
      <div id="experiment-article-sections">

        <!-- First section of the article-->
        <section id="experiment-article-section-1">
          
          <div id="experiment-article-section-1-icon" 
          class="icon">
	    <!-- Enclose the icon image of your lab -->
	    <img src="../images/introduction.jpg" />
	  </div>	
          
          <!-- The heading for the section can be enclosed in a 
          div tag. -->
          <div id="experiment-article-section-1-heading" 
          class="heading">
            Introduction
          </div>

          <!-- Write the section content inside a paragraph 
          element, You can also include images with <img> tag -->
          <div id="experiment-article-section-1-content" 
          class="content" align="justify">	
            <p>
In Experiment 8 we have seen how a protein denatures at high temperature. But temperature is not the sole factor that causes denaturation of proteins. The chemical environment is another factor that is responsible for denaturation of protein. Change of pH of the environment or presence of  denaturing chemical in high concentration in the medium causes denaturation of protein.</p>
<p>
In our daily life we have seen many such examples. Formation of paneer from milk is one of them. The following video shows how egg protein albumin denatures in a highly acidic medium.</p>
<p><br>
<table width="600">
<center>
<embed
  flashvars="file=./intro_video.flv&autostart=true"
  allowfullscreen="true"
  allowscripaccess="always"
  id="player1"
  name="player1"
  src="player.swf"
  width="480"
  height="270"
/>
</center>
</p>
</table>

<!-- <br><i>Adapted from : http://www.youtube.com/watch?v=o2ju5ys3Cxc&feature=related</i><br><br> -->
<p>
In chemical denaturation the secondary bonds holding the protein segments together are disrupted by some chemicals capable of forming equally strong or stronger bonds with the groups holding the conformation together. For disrupting the hydrogen bonds, urea or guanidine hydrochloride are used. At high concentrations of these substances (e.g, 8M urea or 5M guanidine hydrochloride) many proteins adopt a highly unfolded conformation in solution. Proteins of multiple subunits are likely to be separated into their constituent polypeptide chains. Other proteins aggregate upon denaturation in urea or guanidine hydrochloride which is frequently due to the formation of disulphide bridges between sulphydryl groups made accessible by the unfolding of the polypeptide chains.
</p><br>
<p>
This experiment has two parts. In the first part we will observe how helical membrane protein Rhodopsin looses its native structure and unfolds in presence of some denaturing chemicals like urea, SDS, GuHCl and TFA. We will compare their denaturing capability. In the second part of the experiment we will observe how the extend of unfolding depends on the concentration of the chemical. We will try to see the transition between folded and unfolded state and find out the concentration at which the transition takes place.</p>
<br>
        </div>


      </section>

      <!-- Second section of the article-->
      <section id="experiment-article-section-2">
        
        <div id="experiment-article-section-2-icon" 
        class="icon">
	  <!-- Enclose the icon image of your lab. -->
	  <img src="../images/theory.jpg" />
	</div>
				
        <!-- The heading for the section can be enclosed in a 
        div tag. -->
        <div id="experiment-article-section-2-heading" 
        class="heading">
          Theory
        </div>


        <!-- Write the section content inside a paragraph 
        element, we can also include images with <img> tag -->
        <div id="experiment-article-section-2-content" 
        class="content" align="justify">
          <p>Previously, we have already discussed in details in experiment <a href="http://pascal.iiit.ac.in/ccnsb/cd/exp8/" target="_blank">8</a> the causes and the effects of &#34;<a href="http://pascal.iiit.ac.in/ccnsb/cd/exp8/#" target="_blank">protein folding</a>&#34; and &#34;<a href="http://pascal.iiit.ac.in/ccnsb/cd/exp8/#" target="_blank">protein denaturation</a>&#34;, the denatured state and the mechanism behind the same. Continuing from where we had left our study with temperature related effects (thermal denaturation), now we look to extend our search further penetrating into the effects due to chemical denaturants in this experiment. So, without referring again the details of the earlier explained points we limit our discussion to focus in the perspective of the chemical denaturants and their contribution to our cause.</p>

<h2><u><div align="center">Protein Stability</div></u></h2><br>

<p>To be biologically active, proteins must adopt specific folded three-dimensional, tertiary structures. Yet the genetic information for the protein specifies only the primary structure that is the linear sequence of amino acids in the polypeptide backbone. Many purified proteins can spontaneously refold in vitro after being completely unfolded, so the three-dimensional structure must be determined by the primary structure (Anfinsen, 1973). <i>Different conformations of a protein differ only in the angle of rotation about the bonds of the backbone and amino acid side-chains.</i> It may, therefore, appear surprising that a protein folds into a single unique conformation from all the possible rotational conformations available around single bonds in the primary structure of a protein. The question arises as to when a protein folds up to its native conformation, does this structure represent a local or a global energy minimum? The native structure of the protein may correspond to a metastable state with a very long lifetime. If proteins are only <i>metastable</i>, their structures must be grossly different from the most stable one. The polypeptide may adopt a structure corresponding to the global minimum.</p>
<p><i>This led to the idea of a biased random search resulting in faster folding since a proportion of the conformations would be sterically disallowed.</i> A framework model proposed (Baldwin, 1989; Kim and Baldwin, 1990) that the folded structure was formed by packing together pre-existing individual elements of secondary structure which had significant stability in the unfolded protein. It became evident through some experiments that the equilibrium of unfolding of proteins does not always follow a simple two-state model in which only the native and fully unfolded states are significantly populated (Wong and Tanford, 1973). An intermediate compact structure known as the molten globule which is different from the native structure and whose formation is determined mainly by non-specific interactions of amino acid residues with their environment was presented. Specific interactions could direct the folding pathway by stabilizing folded conformations. The best studied example is bovine pancreatic trypsin inhibitor (Creighton, 1978). For this protein it was shown that formation of a disulphide bridge stabilizes secondary structure elements, and the protein refolds by a specific pathway of disulphide bond formation and rearrangement (Directed folding model). <b><i>Since non covalent forces act on the primary structure to cause a protein to fold into a unique conformational structure and then stabilize the native structure against denaturation processes, it is of importance to understand the properties of these forces.</b></i></p><br>


<h2><u><div align="center">Chemical Denaturation</div></u></h2><br>

<p>Protein denaturation has been defined in several ways, for example as a change in solubility (Mirsky, 1941) or by simultaneous changes in <b>chemical</b>, physical and biological properties (Neurath et al., 1944; Langmuir, 1938) under some standard reference set of conditions (Timasheff and Gibbs, 1957).</p>  
<p><b>The term chemical denaturation denotes the response of the native protein to acid, alkali, and a variety of other chemical agents which cause marked changes in the protein structure. Therefore chemical denaturation may be defined in very general terms as any modification of the secondary, tertiary or quaternary structure of the protein molecule, excluding any breaking of covalent bond (Joly, 1965).</b></p>
<p>
Denaturation midpoint is defined as that denaturant concentration (Cm) at which both the folded and unfolded states of a protein are equally populated at equilibrium. It is usually determined by recording CD spectra of protein at various concentrations of the denaturant. The mid point of transition is taken as the point where 50% of the protein’s secondary structure is in conformation 1 and the other 50% is in conformation 2. 
</p>
<br>
<h3><u><div align="justify">Mathematical model behind chemical denaturation</div></u></h3>

<p>In the less expensive technique of equilibrium unfolding, the fractions of folded and unfolded molecules (denoted as p<sub>N</sub> and p<sub>U</sub>, respectively) are measured as the solution conditions are gradually changed from those favoring the native state to those favoring the unfolded state, e.g., by adding a denaturant such as guanidinium hydrochloride or urea. (In equilibrium folding, the reverse process is carried out.) Given that the fractions must sum to one and their ratio must be given by the Boltzmann factor, we have</p>
<br>
<p>
<table>
<tr>
<td align="left">
<img src="form1.gif" style="height:50px; width:150px;">
</td>
</tr>
</table></p>
<p><table>
<tr>
<td align="left">
<img src="form2.gif" style="height:50px; width:150px;">
</td>
</tr>
</table></p>
<p>
Protein stabilities are typically found to vary linearly with the denaturant concentration. A number of models have been proposed to explain this observation prominent among them being the <b>denaturant binding model</b>, <b>solvent-exchange model</b> and the <b>Linear Energy Model</b> (LEM). All of the models assume that only two thermodynamic states are populated/de-populated upon denaturation. They could be extended to interpret more complicated reaction schemes.</p>
<p>The denaturant binding model assumes that there are specific but independent sites on the protein molecule (folded or unfolded) to which the denaturant binds with an effective (average) binding constant k. The equilibrium shifts towards the unfolded state at high denaturant concentrations as it has more binding sites for the denaturant relative to the folded state (&Delta;n). In other words, the increased number of potential sites exposed in the unfolded state is seen as the reason for denaturation transitions. An elementary treatment results in the following functional form:</p>
<table>
<tr>
<td align="left">
<img src="form3.gif"  style="height:50px; width:350px;">
</td>
</tr>
</table>

<p>where &Delta;Gw is the stability of the protein in water and [D] is the denaturant concentration. Thus the analysis of denaturation data with this model requires 7 parameters: &Delta;Gw, &Delta;n, k, and the slopes and intercepts of the folded and unfolded state baselines.</p>
<p>The solvent exchange model (also called the &#39;weak binding model&#39; or &#39;selective solvation&#39;) of Schellman invokes the idea of an equilibrium between the water molecules bound to independent sites on protein and the denaturant molecules in solution. It has the form:</p>
<table>
<tr>
<td align="left">
<img src="form4.gif"  style="height:50px; width:350px;">
</td>
</tr>
</table>

<p>where K is the equilibrium constant for the exchange reaction and X<sub>d</sub> is the mole-fraction of the denaturant in solution. This model tries to answer the question of whether the denaturant molecules actually bind to the protein or they seem to be bound just because denaturants occupy about 20&#45;30 % of the total solution volume at high concentrations used in experiments, i.e. non-specific effects &#45; and hence the term &#39;weak binding&#39;. As in the denaturant-binding model, fitting to this model also requires 7 parameters. One common theme obtained from both these models is that the binding constants (in the molar scale) for urea and guanidinium hydrochloride are small: &#126; 0.2 M<sup>&#45;1</sup> for urea and 0.6M<sup>&#45;1</sup> for GuHCl.</p>
<p>Intuitively, the difference in the number of binding sites between the folded and unfolded states is directly proportional to the differences in the accessible surface area. This forms the basis for the LEM which assumes a simple linear dependence of stability on the denaturant concentration. The resulting slope of the plot of stability versus the denaturant concentration is called the m-value. In pure mathematical terms, m&#45;value is the derivative of the change in stabilization free energy upon the addition of denaturant. However, a strong correlation between the accessible surface area (ASA) exposed upon unfolding, i.e. difference in the ASA between the unfolded and folded state of the studied protein (dASA), and the m-value has been documented by Pace and co-workers. In view of this observation, the m-values are typically interpreted as being proportional to the dASA. There is no physical basis for the LEM and is purely empirical, though it is widely used in interpreting solvent-denaturation data. It has the general form:</p>
<table>
<tr>
<td align="left">
<img src="form5.gif"  style="height:40px; width:300px;">
</td>
</tr>
</table>
<p>
where the slope m is called the &#34;m&#45;value&#34;(> 0 for the above definition) and [D]<sub>1/2</sub> (also called Cm) represents the denaturant concentration at which 50% of the molecules are folded (the denaturation midpoint of the transition, where p<sub>N</sub> &#61; p<sub>U</sub> &#61; 1 / 2).</p>
<p>In practice, the observed experimental data at different denaturant concentrations are fit to a two-state model with this functional form for &Delta;G, together with linear baselines for the folded and unfolded states. The m and [D]<sub>1/2</sub> are two fitting parameters, along with four others for the linear baselines (slope and intercept for each line); in some cases, the slopes are assumed to be zero, giving four fitting parameters in total. The conformational stability &Delta;G can be calculated for any denaturant concentration (including the stability at zero denaturant) from the fitted parameters m and [D]<sub>1/2</sub>. When combined with kinetic data on folding, the m&#45;value can be used to roughly estimate the amount of buried hydrophobic surface in the folding transition state.</p><br><br>

<h3><u><div align="center">Chemical Denaturants</div></u></h3><br>
<p>Chemical Denaturants are classified into numerous categories:<br>
<ul>
<li><b>Acids</b></li>
<p>Acidic protein denaturants include:</p>
	<ul type="square">
		<li>Acetic acid</li>
		<li>Trichloroacetic acid 12% in water</li>
		<li>Sulfosalicylic acid</li>
		<li>Dichloroacetic acid</li>
	</ul>
</li>
<li><b>Solvents</b></li>
<p>Most organic solvents are denaturing, including:</p>
	<ul type="square">
		<li>Ethanol</li>
		<li>Methanol</li>
		<li>Formamide</li>
		<li>Dimethyl Formamide</li>
	</ul>
</li>
<li><b>Cross-linking reagents</b></li>
<p>Cross-linking agents for proteins include:</p>
	<ul type="square">
		<li>Formaldehyde</li>
		<li>Glutaraldehyde</li>
	</ul>
</li>
<li><b>Chaotropic agents</b></li>
<p>Chaotropic agents include:</p>
	<ul type="square">
		<li>Urea 6 &#45; 8 mol/l</li>
		<li>Guanidinium chloride 6 mol/l</li>
		<li>Lithium perchlorate 4.5 mol/l</li>
		<li>Sodium dodecyl sulphate</li>
	</ul>
</li>
<li><b>Disulfide bond reducers</b></li>
<p>Agents that break disulfide bonds by reduction include:</p>
	<ul type="square">
		<li>2&#45;Mercaptoethanol</li>
		<li>Dithiothreitol</li>
		<li>TCEP (tris(2&#45;carboxyethyl)phosphine)</li>
	</ul>
</li>
<li><b>Inorganic salts</b></li>
<p>Lithium bromide, potassium thiocyanate, and sodium iodide.</p>
</li>
</ul>
</p><br><br>

<h3><u><div align="justify">The process of chemical denaturation of a protein:</div></u></h3>
<p>In chemical denaturation the secondary bonds holding the protein segments together are disrupted by some chemicals capable of forming equally strong or stronger bonds with the groups holding the conformation together. For disrupting the hydrogen bonds, urea or guanidine hydrochlorides are used. At high concentrations of these substances (e.g., 8M urea or 5M guanidine hydrochloride) many proteins adopt a highly unfolded conformation in solution. Proteins of multiple subunits are likely to be separated into their constituent polypeptide chains. Other proteins aggregate upon denaturation in urea or guanidine hydrochloride which is frequently due to the formation of disulphide bridges between sulphydryl groups made accessible by the unfolding of the polypeptide chains. Such reactions may be inhibited by the addition of iodoacetate (Friedli, PhD thesis, chapter 3 and 5). Under these conditions, the denatured molecules remain in solution and may revert into native molecules if the denaturing agent is slowly dialysed away. Powerful detergents like SDS disrupt both hydrophobic and hydrogen bonds and effectively solvate the denatured molecule. Beta-Mercaptoethanol and dithiothreitol (DTT) disrupt disulphide bonds and can be used in conjunction with urea or SDS to fully solubilize protein molecules.</p>
<br>
<table>
<tr>
<td align="center">
<img src="pic1.png" style="height:300px; width:500px"><br><text alignment="justify">Ref: The above picture is taken from the website of <a href="http://www.bio.davidson.edu/courses/genomics/method/SDSPAGE/SDSPAGE.html" target="_blank">Department of Biology, Davidson College, Davidson, NC 28036</a><text><br></td></tr>
</table><br>

<h3><u><div align="justify">An example of chemical denaturantion:</div></u></h3>
<p>
Membrane proteins are not soluble. The first work on the folding of membrane proteins was carried out with bacteriorhodopsin (bR) which was successfully refolded into phospholipids via transfer into sodium dodecyl sulfate (SDS) micelles after denaturation by trifluoroacetic acid (TFA). This has established bR as a model system for studying the folding of helical membrane proteins. Kinetic studies of refolding of bR from an SDS-denatured state into lipid/detergent micelles have enabled detection of folding intermediates in its folding pathway.
</p>
<p>
Recently, studies with rhodopsin in phospholipid bicelles revealed a significant decrease of 50% of  its helical content in the presence of urea, but it could not be refolded from such a largely unfolded state.</p>
<p>Different chemical denaturants like 4 M TFA, 8 M GuHCl, 10 M urea, 3% SDS &#43; 8Murea, and 30% SDS have their effect on the secondary structure of Rhodopsin to different extents. Some of them cause aggregation in the unfolded structure. So to study the unfolded structure of Rhodopsin in the presence of chemical denaturants it is important to reinvestigate whether aggregation is taking place or not.</p><br><br>

<h2>Goal of The Experiment</h2>
<p>In this experiment we will see how different denaturing agents are effecting secondary structure of Rhodopsin differently and at the same time we will study the secondary structure of Rhodopsin as a function of the concentration of the denaturant.<br>
</p>

        </div>
      </section>


      <section id="experiment-article-section-3">
        
        <div id="experiment-article-section-3-icon" 
        class="icon">
	  <!-- Enclose the icon image of your lab. -->
	  <img src="../images/objective.jpg" />
	</div>
     
        <div id="experiment-article-section-3-heading" 
        class="heading">
          Objective
        </div>

        <div id="experiment-article-section-3-content" 
        class="content" align="justify">
          <p>The purpose of this lab is to make students familiar with the concept that structure of a protein is highly dependent on the chemical environmental. By performing this experiment a student will also get familiar with the use of CD spectroscopy method to study the effect of chemical denaturant on protein structure. By doing this experiment a student will be able to,</p>
<br><br>
<ol>
	<li>Make the student understand how a chemical environment denatures a protein.</li>
	<li>Have an idea of the effect of different types of chemicals on the protein structure.</li>
	<li>Have an idea of the effect of different concentrations of chemical environment on the protein structure.</li>
	<li>Understand how CD spectroscopy can be used to study unfolding of proteins.</li>
	<li>Observe transition between folded and unfolded state.</li>
	<li>Determine the critical concentration of transition from folded to unfolded structure using CD spectra at different concentrations.</li>
</ol>

        </div>

      </section>


      <section id="experiment-article-section-4">

        <div id="experiment-article-section-4-icon" 
        class="icon">
	  <!-- Enclose the icon image of your lab.-->
	  <img src="../images/simulation.jpg" />
	</div>

        <div id="experiment-article-section-4-heading" 
        class="heading">
          Experiment
        </div>

        <div id="experiment-article-section-4-content" 
        class="content" align="justify">
          <p>
	        <a href="experiment/index.html" target="_blank">Click here</a> to start the experiment. 
	        
            
          </p>
        </div>

      </section>

      <section id="experiment-article-section-5">
   
        <div id="experiment-article-section-5-icon" 
        class="icon">
	  <!-- Enclose the icon image of your lab.-->
	  <img src="../images/manual.jpg" />
	</div>

        <div id="experiment-article-section-5-heading" 
        class="heading">
          Manual
        </div>

        <div id="experiment-article-section-5-content" 
        class="content" align="justify">
          <br><p><h2>1. Video Tutorial</h2></p>
<p>
<param name="movie" value="manual_exp9.swf">
<embed src="manual_exp9.swf">
</embed>
</param></p>
<p>
<h3>click <a href="./manual_exp9.swf" target="_blank">here </a>to ENLARGE.</h3></p>

<br><br>
<p><h2>2. Step-by-step Procedure</h2></p>
<p>The experiment section has three sections – Animation explaining extent of protein unfolding depending on chemical denaturant concentration, part 1 and part 2.
<br></p>
<br><p><b>Section 1</b></p><br>
<p>Click on first section. Observe the animation showing MD simulation of unfolding of a helical protein at different concentration (2M,4M,6M and 8M).
<br>
Click the button BACK to go back to the first page and perform the experiment.
<br></p>
<br><p><b>Section 2</b></p><br>
<p>Click on PART1. A list of chemicals will appear along with their molecular level picture and physiochemical descriptions. Select any number chemicals whose denaturing capability you want to study. Click PROCEED to go to experiment setup.
</p><br>
<p>Steps to follow in this page will appear on top left corner of the page. Follow the instructions. After you are done click on the monitor to see the results.</p>
<br>
<p>Go through the NOTE carefully and then click Result.</p>
<br>
<p>Select the chemical from bottom left panel and the CD spectra. Secondary structure content are listed on the right top panel.</p>
<br>
<p>Once you are done click to “Go to analysis”.</p>
<br>
<p>Answer the questions and submit your answer.<p/>
<br>
<br><p><b>Section 3</b><br></p>
<p>Click on PART2. This will lead to the experimental set-up. Steps to follow in this page will appear on top left corner of the page. Follow the instructions. After you are done click on the monitor to see the results.
<br>
See the CD spectrum of Rhodopsin at different concentrations. Use CONTROLS to see spectrum at different concentrations.  Take your mouse on the graph. Select the MRE value corresponding to 222nm and fill up the TABLE. Click on Plot once you have finished filling up the TABLE. You can come back to this step by clicking Go Back.
<br>
Once you are done click Go To Analysis.</p><br>
<br>

          </div>

        </section>

        <section id="experiment-article-section-6">
      
          <div id="experiment-article-section-6-icon" 
          class="icon">
	    <!-- Enclose the icon image of your lab.-->
	    <img src="../images/quizzes.jpg" />
	  </div>

          <div id="experiment-article-section-6-heading" 
          class="heading">
            Quizzes
          </div>

          <div id="experiment-article-section-6-content" 
          class="content" align="justify">
            
            <p>
<a href="./quiz1/pre_exp_quiz.php">Click here</a> for <b>pre-experiment</b> quiz.<br> <br>
<!--a href="./quiz2/quiz.php">Click here</a> to have a self evaluation for part2.<br-->
</p>

          </div>

        </section>

<!--        <section id="experiment-article-section-7">
	  
          <div id="experiment-article-section-7-icon" 
          class="icon">	-->
	    <!-- Enclose the icon image of your lab. -->
<!--	    <img src="../images/procedure.jpg" />
	  </div>
	
          <div id="experiment-article-section-7-heading" 
          class="heading">
	    Procedure
	  </div>
	
          <div id="experiment-article-section-7-content" 
          class="content">
	    <p>
	      Procedure for the experiment is as follows.
	    </p>
	  </div>
	
        </section>	-->
			
		
        <section id="experiment-article-section-8">
   
          <div id="experiment-article-section-8-icon" 
          class="icon">
	    <!-- Enclose the icon image of your lab.-->
	    <img src="../images/readings.jpg" />
	  </div>

          <div id="experiment-article-section-8-heading" 
          class="heading">
            Further Readings
          </div>

          <div id="experiment-article-section-8-content" 
          class="content" align="justify">
<p>
<ol>
	<li>The use of circular dichroism spectroscopy to study protein folding, form and function;Daniel H. A. Corrêa and Carlos H. I. Ramos; African Journal of Biochemistry Research Vol.3 (5), pp. 164-173, May, 2009</li>
	<li>Characterization of Membrane Protein Non-native States: Extent of Unfolding and Aggregation of Rhodopsin in the Presence of Chemical Denaturants ;Arpana Dutta, Kalyan C. Tirupula, Ulrike Alexiev and Judith Klein-Seetharaman ; Biochemistry 2010, 49, 6317–6328 </li>
	<li>Modern Techniques For Circular Dichroism And Synchrotron Radiation Circular Dichroism Spectroscopy; Bonnie Ann Wallace And Robert William Janes; IOS Press</li>
	<li><a href="http://www.friedli.com/herbs/phytochem/proteins.html#denaturation" target="blank">http://www.friedli.com/herbs/phytochem/proteins.html#denaturation</a></li>

</ol>
</p>
          </div>

        </section>

      </div>


    <!-- An article can have a sidebar that contain related 
    links and additional material (however it is kept optional 
    at this moment) -->
    <aside id="lab-article-sidebar" class="default">
      <!-- put the content that you want to appear in the 
      sidebar -->	
    </aside>


    <!-- Article footer can display related content and 
    additional links -->						
    <footer id="lab-article-footer" class="default">
      <!-- Put the content that you want to appear here -->
    </footer>

  </article>


  <!-- Links to other labs, about us page can be kept the lab 
  footer-->
  <footer id="lab-footer" class="default">
    <!-- Put the content here-->
  </footer>

</div>		

</body>
</html>
